site2/docs/client-libraries-cpp.md

```diff
@@ -129,6 +189,45 @@ Debian packages are created at `pulsar-client-cpp/pkg/deb/BUILD/DEB/`.
 
 ## MacOS
 
+### Compilation
+
+The compilation starts with cloning the Pulsar repository:
```

Comment:
use an ordered list and use a period(.) instead of a colon (:) at the end of each sentence.